uber/site_sections/registration.py

```diff
@@ -348,6 +367,14 @@ def lost_badge(self, session, id):
         session.commit()
         raise HTTPRedirect('index?message={}', 'Badge has been recorded as lost.')
 
+    def convert_group_badge(self, session, id):
```

Comment:
This was fine, but it maybe should be more automatic? Anyone without a badge currently should get a new badge offered to them when the group is declined?